addon/components/nrg-modal/component.js

```diff
@@ -17,7 +18,8 @@ export default Component.extend({
   context: 'body',
   isOpen: false,
   autofocus: true,
-  detachable: true,
+  testing: Ember.testing,
+  detachable: not('testing'),
```

Comment:
I would even simplify it more than this.
```suggestion
  detachable: !Ember.testing,
```